core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -31,19 +31,19 @@ oppia.animation('.oppia-collection-animate-slide', function() {
 });
 
 oppia.controller('CollectionPlayer', [
-  '$scope', '$anchorScroll', '$location', '$http',
+  '$scope', '$anchorScroll', '$location', '$http', '$rootScope',
```

Comment:
Using $rootScope is an antipattern, you'll need to find another way. The suggestion I gave in exploration_summary_tile_directive.html might help.

Reply:
Done